rotkehlchen/tests/unit/test_evm_tx_decoding.py

```diff
@@ -14,9 +15,82 @@
 from rotkehlchen.db.evmtx import DBEvmTx
 from rotkehlchen.db.filtering import EvmTransactionsFilterQuery
 from rotkehlchen.fval import FVal
-from rotkehlchen.types import ChainID, EvmTransaction, Location, Timestamp, deserialize_evm_tx_hash
+from rotkehlchen.types import (
+    ChainID,
+    ChecksumEvmAddress,
+    EvmTransaction,
+    EVMTxHash,
+    Location,
+    Timestamp,
+    deserialize_evm_tx_hash,
+)
 from rotkehlchen.utils.hexbytes import hexstring_to_bytes
 
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.transactions import EthereumTransactions
+    from rotkehlchen.chain.optimism.decoding.decoder import OptimismTransactionDecoder
+    from rotkehlchen.chain.optimism.transactions import OptimismTransactions
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+def _add_transactions_to_db(
```

Comment:
Why move this outside in its own function when it's only used in one place? What do you achieve?

Reply:
It was because I wanted to reuse it but then I noticed that I could consolidate in one test. I left it as an aux function because for me it was easier to read the code without the hustle of the setup